pkg/platform/telemetry/telemetry.go

```diff
@@ -0,0 +1,134 @@
+package telemetry
+
+import (
+	"bytes"
+	"net/http"
+	"time"
+
+	"go.keploy.io/server/pkg/models"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.uber.org/zap"
+)
+
+type Telemetry struct {
+	TelemetryDB DB
```

Comment:
the variable could also just be named `db`

Reply:
Done